test/test_helper.rb

```diff
@@ -51,9 +51,9 @@ class Product < ActiveRecord::Base
   adapter: "mysql2",
   database: "job_iteration_test",
   username: 'root',
-  password: 'root',
   host: host,
 }
+connection_config[:password] = 'root' unless ENV['USING_DEV'] == 1
```

Comment:
Environment variables are always strings, so this is always false, so it doesn't work with dev.

We should check for ENV["CI"] instead, because locally when not using dev, we don't want the password, (if we assume Homebrew MySQL). To test, have one terminal open with `unset USING_DEV`, running `bin/setup` to setup things in the Homebrew MySQL, and one with `dev up`, both should pass `bundle exec rake`.

Reply:
Yes that definitely needs to be a string 🤦‍♀️ 

I think it works with `USING_DEV` (pending it being changed to a string, I tested both scenarios and it works), as long as we make sure to use the password option in `bin/setup` - so then the only time we'd _not_ need a password would be if we were using dev / Railgun.

But I agree that that's a weird way to do it when we could just check CI and _only_ set the password there instead of doing the inverse. 😁